File: HotelWeb/src/app/models/slikaSobe.ts
export interface SlikaSobe {
  id: string;
  naslovSlike: string;
  urlSlike: string;
  opisSlike: string;
  sobaId?: string;
  sobaNaziv?: string;
}
export interface UpsertSlikaSobe {
  naslovSlike?: string | null;
  urlSlike?: File | null;
  opisSlike?: string | null;
  sobaId: string;
}
export interface UpsertNaslovnaSlikaSobe {
  naslovSlike?: string | null;
  urlSlike?: File | null;
  opisSlike?: string | null;
}

File: HotelAPI/Models/Entities/SlikaSobe.cs
namespace HotelAPI.Models.Entities
{
    public class SlikaSobe : Entity
    {

        public required string UrlSlike { get; set; }
        public string? NaslovSlike { get; set; }
        public string? OpisSlike { get; set; }
        public required Guid SobaId { get; set; }
        public Soba Soba { get; set; } = null!;

    }
}
